documentation/docs/configuration/projects.md

```diff
@@ -2,22 +2,42 @@
 sidebar_position: 3
 ---
 
-# Configuration Structure
+# Projects
 
-Configuration files are ordered by `project` in the projects folder. Project folder can only contain:
+A project is a folder containing, which contains specially named
+sub-folders, representing APIs. This API folders then contain another
+layer of folders defining confiugrations. Then finally, this configuration
+folders contain yaml files, specifying what gets deployed.
 
-- configurations
-- or another project(s)
+## APIs
 
-This means, it is possible to group projects into folders.
+To see a list of all supported APIs and folder names, please have a
+look [here](./configTypes_tokenPermissions.md).
 
-Combining projects and configurations in same folder is not supported.
+## Configurations
 
-There is no restriction in the depth of projects tree.
+Configurations consists of two parts:
+- Yaml defining parameters, dependencies, name and template
+- Template file
 
-To get an idea, what are the possible combinations take a look at `cmd/monaco/test-resources/integration-multi-project`.
+### Configuration Yaml
 
-## Config JSON Templates
+Contains basic information about the config to deploy. This includes
+the name of the config, the location of the template file and parameters
+usable in the template file. Parameters can be overwritten based on what
+group or environment is currently deployed.
+
+For more details on the configuration syntax, see [here](yaml_config.md).
+
+### Template File
+
+This file will get evaluated using GO templates and then uploaded to the
+corresponding api endpoint of dynatrace. All parameters defined in the
+configuration yaml are accessible via `{{ .[PARAMETER_NAME] }}`. Since the
+file is basically GO templates, you can use everything GO templates offer.
```

Comment:
You assume the reader knows what a Go Template is, and how it works. 
1. Try to re-write this assuming the reader does not know that
2. Link to the Go templating syntax

Reply:
It might make sense to not mention GO templates at all. Otherwise people will
miss-use it ^^

Reply:
Haha I broke the json verification. I will add it again